.travis.yml

```diff
@@ -22,13 +22,15 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+dist: trusty
+
 addons:
-  postgresql: 9.4
+  postgresql: 9.6
```

Comment:
But then we will have this error when we try to upgrade the production instances. Do you have an idea how to fix it or where it is coming from?